lib/validate-test.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+function validate(title, fn, metadata) {
+	if (metadata.type !== 'test') {
+		if (metadata.exclusive) {
+			return '`only` is only for tests and cannot be used with hooks';
+		}
+
+		if (metadata.failing) {
+			return '`failing` is only for tests and cannot be used with hooks';
+		}
+
+		if (metadata.todo) {
+			return '`todo` is only for documentation of future tests and cannot be used with hooks';
+		}
+	}
```

Comment:
In the issue, it was discussed that skipping hooks should be allowed. Do you think we shouldn't skip them?